packages/web/src/components/gcds-pagination/gcds-pagination.css

```diff
@@ -0,0 +1,199 @@
+:host {
+  display: block;
+
+  ul {
+    padding: 0;
+    width: fit-content;
+    list-style: none;
+
+    li {
+      margin: var(--gcds-spacing-100);
+
+      a {
+        color: var(--var-pagination-default-text);
+        font-weight: 800;
+        font-family: var(--gcds-font-families-body);
+        border-radius: var(--gcds-spacing-50);
+
+        &:hover {
+          background: var(--gcds-pagination-hover-background);
+        }
+
+        &:focus {
+          color: var(--gcds-pagination-focus-text);
+          background-color: var(--gcds-pagination-focus-background);
+
+          box-shadow: 0 0 0 .125rem var(--gcds-pagination-focus-shadow);
+          outline: .1875rem solid var(--gcds-pagination-focus-outline);
+          outline-offset: .125rem;
+        }
+
+        &:active,
+        &[aria-current*=page] {
+          color: var(--gcds-pagination-active-text);
+          background: var(--gcds-pagination-active-background);
+
+          &:focus {
+            color: var(--gcds-pagination-focus-text);
+            background-color: var(--gcds-pagination-focus-background);
+  
+            box-shadow: 0 0 0 .125rem var(--gcds-pagination-focus-shadow);
+            outline: .1875rem solid var(--gcds-pagination-focus-outline);
+            outline-offset: .125rem;
+          }
+        }
+      }
+    }
+
+    @media screen and (max-width: 22.5rem) {
+      li {
+        margin: var(--gcds-spacing-50);
+      }
+    }
+
+    &.gcds-pagination-list,
+    &.gcds-pagination-list-mobile-prevnext {
+      display: flex;
+      flex-direction: row;
+      margin: 0 auto;
+
+      li {
+
+        a,
+        & > span.gcds-pagination-list-ellipses {
+          height: 2.75rem;
+          min-width: 2.75rem;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+
+          &.gcds-pagination-end-button,
+          &.gcds-pagination-end-button-mobile {
+            padding: 0.7rem 0.5rem;
+            width: auto;
+            min-width: auto;
+            height:auto;
+  
+            gcds-icon {
+              &.gcds-pagination-arrow-left {
+                margin-right: var(--gcds-spacing-200);
+              }
+              &.gcds-pagination-arrow-right {
+                margin-left: var(--gcds-spacing-200);
+              }
+            }
+          }
+        }
+
+        &.gcds-pagination-mobile-prevnext {
+          display: none;
+        }
+
+        &.gcds-pagination-list-mobile-ellipses,
+        &.gcds-pagination-list-mobile-ellipses-,
+        &.gcds-pagination-list-mobile-ellipses-360,
+        &.gcds-pagination-list-mobile-ellipses-420,
+        &.gcds-pagination-list-mobile-ellipses-460,
+        &.gcds-pagination-list-mobile-ellipses-488 {
```

Comment:
I think you can summarize all of these classes with:
`&[class^='gcds-pagination-list-mobile-ellipses']`